test/acceptance/stress_test.go

```diff
@@ -131,15 +133,23 @@ func sendTransfersAndAssertTotalBalance(ctx context.Context, network *NetworkHar
 		txHash := contract.TransferInBackground(ctx, ctrlRand.Intn(network.Size()), amount, fromAddress, toAddress)
 		txHashes = append(txHashes, txHash)
 	}
+
+	transferDuration = time.Since(start)
+	startWaiting := time.Now()
+
 	for _, txHash := range txHashes {
 		network.WaitForTransactionInState(ctx, txHash)
 	}
 
+	waitDuration = time.Since(startWaiting)
+
 	for i := 0; i < network.Size(); i++ {
 		actualSum := contract.GetBalance(ctx, i, toAddress)
 		require.EqualValuesf(t, expectedSum, actualSum, "recipient balance did not equal expected balance in node %d", i)
 
 		actualRemainder := contract.GetBalance(ctx, i, fromAddress)
 		require.EqualValuesf(t, benchmarktoken.TOTAL_SUPPLY-expectedSum, actualRemainder, "sender balance did not equal expected balance in node %d", i)
 	}
+
+	return
```

Comment:
I personally find this style less readable than the explicit form `return transferDuration, waitDuration` - had to look for the usage of `waitDuration` before I realized it was implicitly returned.

Reply:
i also don't like the style of a test function returning time values since tomorrow we might want to add more like this ...